ospf6d/ospf6_interface.c

```diff
@@ -1719,8 +1719,12 @@ void ospf6_interface_start(struct ospf6_interface *oi)
 	if (oi->area_id_format == OSPF6_AREA_FMT_UNSET)
 		return;
 
-	if (oi->area)
+	if (oi->area) {
```

Comment:
I don't like this approach.  When a speed change is detected in zebra, zebra creates a if_add callback to all daemons.  This generates a an IF_ADD and an IF_LINK_PARAMS update to the daemons.  Both of which are receivable and as in ospf, the cost can be recalculated without having to force an interface down then up.  Please use the approach taken in ospfd and isisd.

Reply:
Okay. But looks `IF_LINK_PARAMS` is for mpls, not for changed speed. In this case, only need recalculate cost. I have re-pushed one.